1.8.9-Forge/src/main/java/net/ccbluex/liquidbounce/features/command/commands/AutoSettingsCommand.kt

```diff
@@ -70,4 +63,39 @@ class AutoSettingsCommand : Command("autosettings", arrayOf("setting", "settings
             return
         }
     }
+
+    override fun tabComplete(args: Array<String>): List<String> {
+        if (args.isEmpty()) return emptyList()
+
+        return when (args.size) {
+            1 -> listOf("list", "load").filter { it.startsWith(args[0], true) }
+            2 -> {
+                if (args[0].equals("load", true)) {
+                    return this.getSettings()
+                        .map { it.asJsonObject["name"].asString }
+                        .filter { it.startsWith(args[1], true) }
+                }
+                return emptyList()
+            }
+            else -> emptyList()
+        }
+    }
+
+    private fun getSettings(): JsonArray {
+        if (this.autoSettings != null) return this.autoSettings!!
+
+        try {
+            val listUrl = "https://api.github.com/repos/CCBlueX/FileCloud/contents/LiquidBounce/autosettings"
+            val listElement = JsonParser().parse(NetworkUtils.readContent(listUrl))
```

Comment:
Wenn sich `NetworkUtils.readContent()` nicht vernünftig verbinden kann, sind das 10s, die der Client laggt. Viele Nutzer könnten da denken der Client sei abgeschmiert. Vielleicht solltest du das in sowas wie nem Thread laufen lassen 🤔